content/docs/reference/spec/migration/buildpack-api-0.4-0.5.md

```diff
@@ -0,0 +1,33 @@
++++
+title="Buildpack API 0.4 -> 0.5"
++++
+
+<!--more-->
+
+This guide is most relevant to buildpack authors. 
+
+See the [spec release](https://github.com/buildpacks/spec/releases/tag/buildpack%2Fv0.5) for buildpack API 0.5 for the full list of changes and further details.
+
+### Stack ID
+The buildpack may now indicate compatibility with any stack by specifying the special value `*`.
+
+### Character restrictions for process types
+For each process in `launch.toml`, `type` is now restricted to only contain numbers, letters, and the characters ., _, and -, so symlinks on both linux and windows filesystems can be created.
```

Comment:
```suggestion
For each process in `launch.toml`, `type` is now restricted to only contain numbers, letters, and the characters `.`, `_`, and `-`, so symlinks on both linux and windows filesystems can be created.
```

Reply:
Fixed as part of https://github.com/buildpacks/docs/pull/279/commits/030d158ab700a4ad8b91f90b8465f4b085408655.